WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Welcome/MigrationWelcomeViewController.swift

```diff
@@ -66,7 +82,11 @@ final class MigrationWelcomeViewController: UIViewController {
 
     private func setupNavigationBar() {
         self.navigationItem.rightBarButtonItem = UIBarButtonItem(email: viewModel.gravatarEmail) { [weak self] () -> Void in
-            self?.viewModel.configuration.actionsConfiguration.secondaryHandler?()
+            guard let self else {
+                return
+            }
+            self.tracker.track(.welcomeScreenAvatarTapped)
+            self.viewModel.configuration.actionsConfiguration.secondaryHandler?()
```

Comment:
As I explained in this [comment](https://github.com/wordpress-mobile/WordPress-iOS/pull/19711/files#r1038388906), the "Me" button also calls `configuration.secondaryHandler?()`